<launch>
    <!-- <rosparam file="$(find fusion_slam)/config/lio.yaml" command="load"/> -->
    <arg name="config_path" default="$(find fusion_slam)/config/lio.yaml"/>
    <node pkg="fusion_slam" type="lio_node" name="lio_node" output="screen">
        <param name="config_path" value="$(arg config_path)"/>
    </node>
    launch-prefix="gdb -ex run --args"
</launch>